.github/workflows/cd-push-to-pypi.yaml

```diff
@@ -9,19 +9,19 @@ jobs:
   poetry-publish:
     runs-on: ubuntu-latest
     environment: Pypi Publish
+    env:
+      python-version: "3.9"
     steps:
       - uses: actions/checkout@v3
 
-      - name: Set up Python 3.9
-        uses: actions/setup-python@v4
+      - name: Setup Python ${{ inputs.python-version }} Environment
```

Comment:
Inputs?  I don't think that's correct. This should be env

Reply:
Will fix, but didn't get a chance to test this one.